benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -157,6 +157,12 @@ protected BenchmarkServiceFutureStub build(io.grpc.Channel channel,
     }
   }
 
+  public static abstract class AbstractBenchmarkService implements BenchmarkService, io.grpc.stub.BindableService {
```

Comment:
Ok, I can get behind this change.

For selfish reasons I would prefer to name this class BaseFooService, since that is what we name the analog internally.  I am probably going to be addding changes that add "unimplemented" method overrides to this class in a soon to be created PR.